translation/zh_CN.po

```diff
@@ -0,0 +1,1926 @@
+"periodically wiped."
+msgstr ""
+"当挤出头处于非活跃状态时用到的温度差。启用一个全高度的环边，纯用于定期给喷嘴"
+"擦拭。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1247 xs/src/libslic3r/PrintConfig.cpp:1256
+msgid "Start G-code"
+msgstr "起始G代码"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1248
+msgid ""
+"This start procedure is inserted at the beginning, after bed has reached the "
+"target temperature and extruder just started heating, and before extruder "
+"has finished heating. If Slic3r detects M104, M109, M140 or M190 in your "
+"custom codes, such commands will not be prepended automatically so you're "
+"free to customize the order of heating commands and other custom actions. "
+"Note that you can use placeholder variables for all Slic3r settings, so you "
+"can put a \"M109 S[first_layer_temperature]\" command wherever you want."
+msgstr ""
+"该段代码放在G代码开头，在热床达到目标温度、挤出头开始加热后，挤出头完成加热前"
+"插入。如果其中不包含有M104, M109, M140 或 M190，这段指令就会在Slic3r自动放置"
+"的M指令之后；否则Slic3r将不会自动生成M指令，这样你可以自由定义加热命令和其他"
+"自定义动作的顺序。注意可以使用占位符变量替代所有Slic3r参数，所以你可以把一"
+"个“M109 S[ first_layer_temperature ]”命令放在任意处。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1257
+msgid ""
+"This start procedure is inserted at the beginning, after any printer start "
+"gcode. This is used to override settings for a specific filament. If Slic3r "
+"detects M104, M109, M140 or M190 in your custom codes, such commands will "
+"not be prepended automatically so you're free to customize the order of "
+"heating commands and other custom actions. Note that you can use placeholder "
+"variables for all Slic3r settings, so you can put a \"M109 "
+"S[first_layer_temperature]\" command wherever you want. If you have multiple "
+"extruders, the gcode is processed in extruder order."
+msgstr ""
+"该段代码放在G代码开头，在所有打印机起始G代码后插入。用于覆盖指定材料的参数。"
+"该段代码放在G代码开头，在热床达到目标温度、挤出头开始加热后，挤出头完成加热前"
+"插入。如果其中不包含有M104, M109, M140 或 M190，这段指令就会在Slic3r自动放置"
+"的M指令之后；否则Slic3r将不会自动生成M指令，这样你可以自由定义加热命令和其他"
+"自定义动作的顺序。注意可以使用占位符变量替代所有Slic3r参数，所以你可以把一"
+"个“M109 S[ first_layer_temperature ]”命令放在任意处。如果你有多个挤出头，G代"
+"码将按挤出头顺序处理。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1269
+msgid "Generate support material"
+msgstr "生成支撑材料"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1271
+msgid "Enable support material generation."
+msgstr "启用支撑材料生成功能。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1276
+msgid "Pattern angle"
+msgstr "样式角度"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1278
+msgid ""
+"Use this setting to rotate the support material pattern on the horizontal "
+"plane."
+msgstr "使用该参数对支撑材料的样子在水平面上进行旋转。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1286
+msgid "Support on build plate only"
+msgstr "仅在热床上生成支撑"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1288
+msgid ""
+"Only create support if it lies on a build plate. Don't create support on a "
+"print."
+msgstr "仅在热床上生成支撑，不在打印出来的物体上生成。"
+
+#: xs/src/libslic3r/PrintConfig.cpp:1294
+#, fuzzy
+msgid "Contact Z distance"
+msgstr "接触Z距离"
+
+# How come " Setting this to 0 will also prevent *speed* for the first object layer"?
```

Comment:
Bridging flow has its own special speed category. Setting support distance to 0 means that the first object layer over the interface will no longer be considered a bridge.